src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -109,6 +109,15 @@ default InteractionResult onItemUseFirst(ItemStack stack, UseOnContext context)
         return InteractionResult.PASS;
     }
 
+    /**
+     * @param entity    The entity using the item
+     * @param itemStack The last {@link ItemStack} held before it stopped being used.
+     */
+    default void stopUsingItem(net.minecraft.world.entity.LivingEntity entity, ItemStack itemStack)
```

Comment:
No need for fully qualified names here.
The parameter name for `ItemStack` parameters is usually just `stack`